app/src/pages/HomePage.tsx

```diff
@@ -106,7 +106,7 @@ export function HomePage() {
               </CardDescription>
             </div>
 
-            <Button variant="outline" asChild>
+            <Button className="mt-5 sm:mt-5 lg:mt-5" variant="outline" asChild>
```

Comment:
I think we could do this, right?

```suggestion
            <Button className="mt-5 lg:mt-0" variant="outline" asChild>
```

Reply:
OK, Sir  i will make 'lg: mt-0' , and  Sir don't you think there can be a small margin between text and and Create environment button ?




![Screenshot from 2024-06-28 22-49-02](https://github.com/ssoready/ssoready/assets/162329540/eb8a78f8-e593-4546-b9e6-0cde8fa24d31)

Reply:
And Good to hear from your side,

For the sake of not messing up things, I need to confirm a few things from your side.

You are saying that I need to remove the updation of Page.tsx from this pr and then Just Push the updates of HomePage.tsx in this pr . And I need to create new Pr for the updates in the code of Page.tsx & mention the #34 issue number in that pr also?

Let me know if I am right or wrong. If wrong, then please correct me there

Thanks

Reply:
I think you understand me, yeah. :) To put it plainly:

1. Include the suggested change to `lg:mt-0`
2. Remove changes to `Page.tsx`
3. I'll merge the PR.

On your other point, I totally agree a bit of spacing would be good. I'm open to doing that in this PR or later. Up to you. Probably a `mr-2` on the paragraph of text would do the trick?